examples/street_canyon_radiosity.ipynb

```diff
@@ -0,0 +1,523 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Street canyon example in Radiosity\n",
+    "\n",
+    "This example shows the process of simulating a geometry such as a street canyon while applying different properties to the wall boundaries. The setup of the canyon is comparable to **Sound propagation in street canyons: Comparison between\n",
+    "diffusely and geometrically reflecting boundaries** by Kang published in 2000.\n",
+    "\n",
+    "Paper: https://doi.org/10.1121/1.428580"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "First, import all of our dependencies."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pyfar as pf\n",
+    "import sofar as sf\n",
+    "import spharpy as sph\n",
+    "import sparrowpy as sp\n",
+    "import matplotlib.pyplot as plt\n",
+    "import os\n",
+    "\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Define the street canyon and set general parameters for the simulation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# canyon boundary dimensions in meters 120 20 18\n",
+    "length = 35  # X\n",
+    "width = 10  # Y\n",
+    "height = 4  # Z  (different heights were used by Kang)\n",
+    "\n",
+    "# simulation parameters\n",
+    "patch_size = 4  # small values increase computation time by a lot\n",
+    "etc_duration = 1\n",
+    "etc_time_resolution = 1 / 1000\n",
+    "max_reflection_order = 90\n",
```

Comment:
```suggestion
    "etc_time_resolution = 1 / 100\n",
    "max_reflection_order = 50\n",
```
I would reduce it, otherwise the example will take too much time